.github/workflows/run-unit-test.yml

```diff
@@ -6,6 +6,11 @@ jobs:
   unit-test:
     runs-on: ubuntu-latest
     steps:
+      - name: install required libraries
```

Comment:
even though we're now running the test using a docker container, the CI complains about a missing `pcap.h` file. This solves the issue